ambillion-ui/src/components/products/productsList.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import { Icon } from '@iconify/react';
+import { TableFilter } from 'components/common/table/tableFilter';
+import DataTable from 'react-data-table-component';
+import { productTableColumns, ActionsColumn, ProductDataRow } from 'utils/table/columns';
+import { dummyProductTableData } from 'utils/table/data';
+import { ConfirmationModal } from 'components/common/modal/confirmationModal';
+import { AddProductModal } from 'components/common/modal/addProductModal';
+
+const customStyles = {
```

Comment:
Check if same Custom styles is used in users table if yes then make it reusable.